FishBun/src/main/java/com/sangcomz/fishbun/util/ImageRotateTransformation.kt

```diff
@@ -0,0 +1,54 @@
+package com.sangcomz.fishbun.util
+
+import android.content.ContentResolver
+import android.database.Cursor
+import android.graphics.Bitmap
+import android.graphics.Matrix
+import android.net.Uri
+import android.provider.MediaStore
+import android.widget.ImageView
+import androidx.exifinterface.media.ExifInterface
+import com.squareup.picasso.Transformation
+
+class ImageRotateTransformation(private val imageView : ImageView, private val loadUrl : Uri) : Transformation {
+    private val contentResolver : ContentResolver = imageView.context.contentResolver;
+
+    override fun key(): String = "imageRotateTransformation";
+
+    override fun transform(source: Bitmap): Bitmap {
+        val realPath: String = getRealPathFromUri(contentResolver, loadUrl)
+        if (realPath == "") return source
```

Comment:
Recommend the logic of an empty string as suggestion
```suggestion
        if (realPath.isEmpty()) return source
```